app/models/grda_warehouse/version.rb

```diff
@@ -0,0 +1,11 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module GrdaWarehouse
+  class Version < GrdaWarehouseBase
+    include PaperTrail::VersionConcern
```

Comment:
This makes sense to me, just noting that it uses a different pattern than the health db https://github.com/greenriver/hmis-warehouse/blob/0d11a228fcb9926a63f8e2e1987d1a6d9ccc67a1/app/models/health/health_version.rb#L9-L10 which inherits from a shared `GrPaperTrail::Version` class to similar effect

Reply:
Yeah I suspect calling establish_connection like that isn't supported or at least it's not well documented. So probably better to avoid it if possible